plans/session-recording.fmf

```diff
@@ -23,6 +23,12 @@ adjust:
       - how: shell
         script: dnf -y upgrade python3-specfile || dnf -y install python3-specfile
 
+  - when: initiator == packit
+    because: "Fix Dubious Ownership, add a safe directory exception for CI"
+    prepare+:
+      - how: shell
+        script: 'git config --global safe.directory /tmp/*'
+
```

Comment:
And this won't help with requre test suite, cause it's the **tests** that are imported in requre, not the **plans**. So you'd need to add this to the requre plan (`packit-integration` iirc) anyways

Reply:
I think the reason why the git causes issues only in requre is that requre pulls in full git in the tests, it probably has some policies / default settings that trigger this. I had similar issue when trying to do the reverse-dependency tests with importing plans in ogr. 

Maybe it would be for the best to keep this setting here and import whole plan instead of just tests from requre, but I guess that would involve a bit more "funny" changes to both tmt plans/tests in here and requre.